Sources/SDFCore/MacroInitializerRewriter.swift

```diff
@@ -1,3 +1,4 @@
+import SwiftOperators
```

Comment:
This comment is also part of a [comment below](https://github.com/fdiaz/SafeDI-Formatter/pull/8/files#diff-142e6f5ebc15f3fa17138a4c7c6f14d9aa35e0a512b080815787e9f0e5f079e4R135).

SwiftSyntax can only know about `SequenceExprSyntax` types, which is a high level type that doesn't include information about what type of expression it is. For example, if we want to know about whether an `SequenceExprSyntax` is an infix, prefix or postfix operator we can't get that directly from "pure" SwiftSyntax, and we need to "fold" the `SequenceExprSyntax`.

Read [this](https://swiftpackageindex.com/swiftlang/swift-syntax/600.0.0/documentation/swiftsyntax/infixoperatorexprsyntax) for a bit more too.